FRC1710-2019/src/main/java/frc/robot/ClawControl.java

```diff
@@ -10,13 +10,23 @@
 
 package frc.robot;
 
+import java.beans.Encoder;
+
 import edu.wpi.first.wpilibj.DoubleSolenoid;
 import edu.wpi.first.wpilibj.Joystick;
+import edu.wpi.first.wpilibj.Joystick.ButtonType;
+import edu.wpi.first.wpilibj.buttons.Button;
+import edu.wpi.first.wpilibj.buttons.JoystickButton;
+import edu.wpi.first.wpilibj.command.Command;
+import edu.wpi.first.wpilibj.Timer;
 
 public class ClawControl {
 
     public static DoubleSolenoid LPiston, RPiston;
     public static Joystick driveStick = new Joystick(0);
+    public static double ReversePower = .2;
+    public static double TimerDelay = 1;
```

Comment:
what is the timer reading in? miliseconds, seconds?

Reply:
A comment like
`public static double TimerDelay = 1; // seconds `
But better would be to add it to the name so it is clear
`public static final double TIMER_DELAY_SECONDS = 1;`

Reply:
To add on to this, will these values ever change? If not it might be worth making them final, so they become constants for this class. See this example here: https://stackoverflow.com/a/66076